packages/libcap.rb

```diff
@@ -24,22 +16,25 @@ class Libcap < Package
   def self.patch
     # add includes option to make it work with gperf-3.1
     system 'sed -e "/gperf --/s/gperf --/gperf --includes --/" -e "/gperf --/s/cap_lookup_name(/cap_dummy(/" -i libcap/Makefile'
-    # change the path to ld
-    system "sed -i 's,/usr/bin/ld,#{CREW_PREFIX}/bin/ld,g' Make.Rules"
+
+    # change the path to mold
+    system "sed -i 's,/usr/bin/ld,#{CREW_PREFIX}/bin/mold,g' Make.Rules"
```

Comment:
```make
ifndef lib
lib=$(shell ldd /usr/bin/ld|grep -E "ld-linux|ld.so"|cut -d/ -f2)
endif
```
We should be able to bypass this by just overriding the lib variable to LIB_SUFFIX.